tests/integration/test_cogs_rates.py

```diff
@@ -1,11 +1,22 @@
 import os
 
+import discord
+import discord.ext.test as dpytest
 import pytest
 
-from poglink.cogs.rates import Rates
-from poglink.error import RatesFetchError, RatesProcessError, RatesWriteError
+from poglink.cogs.rates import EMBED_IMAGE, Rates
+from poglink.error import RatesFetchError, RatesProcessError
 from poglink.models.rates import RatesDiffItem
 
+SAMPLE_DYNAMICCONFIG_PATH_1 = "tests/data/dynamicconfig-1.ini"
+SAMPLE_DYNAMICCONFIG_PATH_2 = "tests/data/dynamicconfig-2.ini"
+
+with open(SAMPLE_DYNAMICCONFIG_PATH_1) as f:
+    SAMPLE_DYNAMICCONFIG_1 = f.read()
+
+with open(SAMPLE_DYNAMICCONFIG_PATH_2) as f:
+    SAMPLE_DYNAMICCONFIG_2 = f.read()
```

Comment:
These already exist as fixtures, but you can't (easily) supply fixtures as values to `pytest.mark.parameterize()` as you'll see below. So I just read them in as regular variables. 

Might be a better way, but I didn't want to get even more off the beaten path than we already have here.